messaging/src/main/java/org/axonframework/common/ReflectionUtils.java

```diff
@@ -247,6 +247,18 @@ public static Method methodOf(Class<?> clazz, String methodName, Class<?>... par
         return primitiveWrapperType;
     }
 
+    /**
+     * Returns the boxed wrapper type for the given {@code type} if it is primitive.
+     *
+     * @param type The type to return boxed wrapper type for
```

Comment:
Minor: I'd write this line as `@param type a {@link Type} to return boxed wrapper type for`